lib/modules/@apostrophecms/modals/src/apos/components/AposModalToolbar.vue

```diff
@@ -30,7 +30,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-  @import '../../../../ui/scss/_mixins';
+  @import 'Modules/@apostrophecms/ui/scss/_mixins';
```

Comment:
Might not need this import since it's included in the `imports.scss` file already.